src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,59 @@
+package duke.task;
+
+import duke.DukeException;
+import duke.task.Task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private List<Task> tasks;
```

Comment:
I like your naming of list objects as plural because it conforms to code quality guidelines 😺